mlir/lib/Dialect/X86Vector/Transforms/LegalizeForLLVMExport.cpp

```diff
@@ -182,79 +100,34 @@ struct DotOpConversion : public ConvertOpToLLVMPattern<DotOp> {
   LogicalResult
   matchAndRewrite(DotOp op, OpAdaptor adaptor,
                   ConversionPatternRewriter &rewriter) const override {
-    auto opType = adaptor.getA().getType();
-    Type llvmIntType = IntegerType::get(&getTypeConverter()->getContext(), 8);
+    Type llvmIntType = rewriter.getIntegerType(8);
```

Comment:
[nit] `llvmByteType` or `llvmInt8Type`? I guess at this level you can assume that byte is 8 bits?

Reply:
Simplified whole creation.